webauthn4j-util/src/main/java/com/webauthn4j/util/KeyUtil.java

```diff
@@ -53,13 +55,13 @@ public static KeyPairGenerator createECKeyPairGenerator() {
 
     public static KeyPair createECKeyPair(byte[] seed, ECParameterSpec ecParameterSpec) {
         KeyPairGenerator keyPairGenerator = createECKeyPairGenerator();
-        SecureRandom random = null;
+        SecureRandom random;
         try {
             if (seed != null) {
                 random = SecureRandom.getInstance("SHA1PRNG"); // to make it deterministic
                 random.setSeed(seed);
             } else {
-                random = SecureRandom.getInstanceStrong();
```

Comment:
Since `SecureRandom.getInstanceStrong()` returns `SecureRandom` with `NativePRNGBlocking`, it makes large overhead even when `SecureRandom.nextBytes()` is called.
I don't see value on using `SecureRandom.getInstanceStrong()` here.